core/cli/src/tasks.ts

```diff
@@ -64,6 +65,7 @@ ${availableHooks}`
   }
 
   await checkInstall(logger, config)
+  await runInit(logger, config)
```

Comment:
**question**: why doesn't `runInit` run before `checkInstall`? initialisation logic would have implicitly run first previously as `checkInstall` loads the plugins

Reply:
good point!